2024-03-22 17:06:52.458 IST [14508] FATAL:  lock file "postmaster.pid" already exists
2024-03-22 17:06:52.458 IST [14508] HINT:  Is another postmaster (PID 14456) running in data directory "/data/data/com.termux/files/home/nodeLogin/db"?
2024-03-22 17:07:00.783 IST [14534] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 17:07:00.785 IST [14534] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 17:07:00.791 IST [14534] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 17:07:00.800 IST [14539] LOG:  database system was interrupted; last known up at 2024-03-22 17:06:19 IST
2024-03-22 17:07:00.955 IST [14539] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-22 17:07:00.969 IST [14539] LOG:  redo starts at 0/153AF48
2024-03-22 17:07:00.969 IST [14539] LOG:  invalid record length at 0/153AF80: expected at least 24, got 0
2024-03-22 17:07:00.969 IST [14539] LOG:  redo done at 0/153AF48 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-22 17:07:00.975 IST [14537] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-22 17:07:00.995 IST [14537] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.002 s, total=0.023 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/153AF80, redo lsn=0/153AF80
2024-03-22 17:07:01.003 IST [14534] LOG:  database system is ready to accept connections
2024-03-22 17:09:35.996 IST [15239] FATAL:  role "postgres" does not exist
2024-03-22 17:09:45.739 IST [15249] FATAL:  database "pistgres" does not exist
2024-03-22 17:09:56.731 IST [15267] FATAL:  lock file "postmaster.pid" already exists
2024-03-22 17:09:56.731 IST [15267] HINT:  Is another postmaster (PID 14534) running in data directory "/data/data/com.termux/files/home/nodeLogin/db"?
2024-03-22 17:11:11.634 IST [15276] ERROR:  role "sankar" already exists
2024-03-22 17:11:11.634 IST [15276] STATEMENT:  CREATE ROLE sankar WITH LOGIN PASSWORD 'sankarraul';
2024-03-22 17:12:00.997 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:12:01.426 IST [14537] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.409 s, sync=0.006 s, total=0.429 s; sync files=5, longest=0.004 s, average=0.002 s; distance=3 kB, estimate=3 kB; lsn=0/153BD58, redo lsn=0/153BD20
2024-03-22 17:12:44.965 IST [15276] ERROR:  role "sankar" already exists
2024-03-22 17:12:44.965 IST [15276] STATEMENT:  CREATE ROLE sankar WITH LOGIN PASSWORD 'sankarraul';
2024-03-22 17:16:13.170 IST [15850] FATAL:  database "nodelogin" does not exist
2024-03-22 17:16:22.362 IST [15857] FATAL:  database "nodelogin" does not exist
2024-03-22 17:17:00.473 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:17:01.118 IST [14537] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.615 s, sync=0.009 s, total=0.646 s; sync files=5, longest=0.004 s, average=0.002 s; distance=1 kB, estimate=3 kB; lsn=0/153C3F0, redo lsn=0/153C3B8
2024-03-22 17:17:41.134 IST [16252] ERROR:  permission denied for table users
2024-03-22 17:17:41.134 IST [16252] STATEMENT:  SELECT * FROM users WHERE email = $1
2024-03-22 17:19:15.605 IST [16513] ERROR:  value too long for type character varying(50)
2024-03-22 17:19:15.605 IST [16513] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:21:32.943 IST [17153] ERROR:  value too long for type character varying(50)
2024-03-22 17:21:32.943 IST [17153] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:22:00.202 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:22:00.648 IST [14537] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.417 s, sync=0.007 s, total=0.447 s; sync files=5, longest=0.004 s, average=0.002 s; distance=2 kB, estimate=3 kB; lsn=0/153CC90, redo lsn=0/153CC58
2024-03-22 17:22:07.934 IST [17195] FATAL:  database "users" does not exist
2024-03-22 17:24:34.013 IST [17209] ERROR:  permission denied for sequence users_id_seq
2024-03-22 17:24:34.013 IST [17209] STATEMENT:  INSERT INTO users (name, email, password)
	VALUES ('Sankar Raul', 'raulsankar99@gmail.com', 'sankar1');
2024-03-22 17:25:16.897 IST [17209] ERROR:  permission denied for sequence users_id_seq
2024-03-22 17:25:16.897 IST [17209] STATEMENT:  INSERT INTO users (name, email, password)
	VALUES ('Sankar Raul', 'raulsankar99@gmail.com', 'sankar1');
2024-03-22 17:26:14.167 IST [17209] ERROR:  permission denied for sequence users_id_seq
2024-03-22 17:26:14.167 IST [17209] STATEMENT:  GRANT USAGE, SELECT ON SEQUENCE users_id_seq TO sankar;
2024-03-22 17:26:42.458 IST [17209] WARNING:  no privileges were granted for "users_id_seq"
2024-03-22 17:26:57.264 IST [17209] ERROR:  syntax error at or near "ls" at character 1
2024-03-22 17:26:57.264 IST [17209] STATEMENT:  ls
	;
2024-03-22 17:27:00.749 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:27:01.384 IST [14537] LOG:  checkpoint complete: wrote 7 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.615 s, sync=0.003 s, total=0.636 s; sync files=7, longest=0.003 s, average=0.001 s; distance=3 kB, estimate=3 kB; lsn=0/153D928, redo lsn=0/153D8F0
2024-03-22 17:27:23.324 IST [17641] ERROR:  value too long for type character varying(50)
2024-03-22 17:27:23.324 IST [17641] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:30:58.332 IST [18033] FATAL:  role "sa" does not exist
2024-03-22 17:32:00.481 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:32:00.806 IST [14537] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.308 s, sync=0.005 s, total=0.325 s; sync files=4, longest=0.003 s, average=0.002 s; distance=2 kB, estimate=3 kB; lsn=0/153E508, redo lsn=0/153E4D0
2024-03-22 17:32:19.542 IST [18099] ERROR:  value too long for type character varying(50)
2024-03-22 17:32:19.542 IST [18099] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:39:14.183 IST [18517] ERROR:  value too long for type character varying(50)
2024-03-22 17:39:14.183 IST [18517] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:41:33.397 IST [19201] ERROR:  value too long for type character varying(50)
2024-03-22 17:41:33.397 IST [19201] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:41:38.393 IST [19216] ERROR:  value too long for type character varying(50)
2024-03-22 17:41:38.393 IST [19216] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:41:39.632 IST [19221] ERROR:  value too long for type character varying(50)
2024-03-22 17:41:39.632 IST [19221] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:41:39.809 IST [19229] ERROR:  value too long for type character varying(50)
2024-03-22 17:41:39.809 IST [19229] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:41:39.836 IST [19231] ERROR:  value too long for type character varying(50)
2024-03-22 17:41:39.836 IST [19231] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:42:01.005 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:42:01.151 IST [14537] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.117 s, sync=0.004 s, total=0.147 s; sync files=2, longest=0.003 s, average=0.002 s; distance=0 kB, estimate=2 kB; lsn=0/153E890, redo lsn=0/153E858
2024-03-22 17:42:53.908 IST [18039] ERROR:  syntax error at or near "`" at character 1
2024-03-22 17:42:53.908 IST [18039] STATEMENT:  `INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
	select * from users;
2024-03-22 17:43:50.678 IST [18039] ERROR:  syntax error at or near "`" at character 1
2024-03-22 17:43:50.678 IST [18039] STATEMENT:  `INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
	select * from users;
2024-03-22 17:45:00.930 IST [19575] ERROR:  value too long for type character varying(50)
2024-03-22 17:45:00.930 IST [19575] STATEMENT:  INSERT INTO users (name,email,password) VALUES ($1,$2,$3)
2024-03-22 17:47:01.161 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:47:01.407 IST [14537] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.214 s, sync=0.007 s, total=0.247 s; sync files=3, longest=0.005 s, average=0.003 s; distance=0 kB, estimate=2 kB; lsn=0/153EC10, redo lsn=0/153EBD8
2024-03-22 17:50:28.355 IST [19481] ERROR:  must be owner of table users
2024-03-22 17:50:28.355 IST [19481] STATEMENT:  ALTER TABLE users
	ALTER COLUMN password SET DATA TYPE text;
2024-03-22 17:56:03.294 IST [19481] ERROR:  syntax error at or near "*" at character 8
2024-03-22 17:56:03.294 IST [19481] STATEMENT:  delete *  from users;
2024-03-22 17:56:21.316 IST [19481] ERROR:  syntax error at or near ";" at character 18
2024-03-22 17:56:21.316 IST [19481] STATEMENT:  alter table users;
2024-03-22 17:56:31.993 IST [19481] ERROR:  syntax error at or near ";" at character 13
2024-03-22 17:56:31.993 IST [19481] STATEMENT:  alter table
	;
2024-03-22 17:57:01.605 IST [14537] LOG:  checkpoint starting: time
2024-03-22 17:57:04.376 IST [14537] LOG:  checkpoint complete: wrote 28 buffers (0.2%); 0 WAL file(s) added, 0 removed, 0 recycled; write=2.733 s, sync=0.021 s, total=2.772 s; sync files=25, longest=0.004 s, average=0.001 s; distance=86 kB, estimate=86 kB; lsn=0/1554460, redo lsn=0/1554428
2024-03-22 17:59:17.790 IST [14534] LOG:  received fast shutdown request
2024-03-22 17:59:17.803 IST [14534] LOG:  aborting any active transactions
2024-03-22 17:59:17.803 IST [19481] FATAL:  terminating connection due to administrator command
2024-03-22 17:59:17.803 IST [20066] FATAL:  terminating connection due to administrator command
2024-03-22 17:59:17.804 IST [15710] FATAL:  terminating connection due to administrator command
2024-03-22 17:59:17.811 IST [14534] LOG:  background worker "logical replication launcher" (PID 14542) exited with exit code 1
2024-03-22 17:59:17.813 IST [14537] LOG:  shutting down
2024-03-22 17:59:17.816 IST [14537] LOG:  checkpoint starting: shutdown immediate
2024-03-22 17:59:17.837 IST [14537] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.006 s, sync=0.004 s, total=0.024 s; sync files=3, longest=0.003 s, average=0.002 s; distance=1 kB, estimate=77 kB; lsn=0/15548C0, redo lsn=0/15548C0
2024-03-22 17:59:17.853 IST [14534] LOG:  database system is shut down
2024-03-22 18:02:03.403 IST [21575] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 18:02:03.406 IST [21575] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 18:02:03.411 IST [21575] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 18:02:03.423 IST [21580] LOG:  database system was shut down at 2024-03-22 17:59:17 IST
2024-03-22 18:02:03.451 IST [21575] LOG:  database system is ready to accept connections
2024-03-22 18:02:40.474 IST [21575] LOG:  received fast shutdown request
2024-03-22 18:02:40.478 IST [21575] LOG:  aborting any active transactions
2024-03-22 18:02:40.485 IST [21575] LOG:  background worker "logical replication launcher" (PID 21583) exited with exit code 1
2024-03-22 18:02:40.485 IST [21578] LOG:  shutting down
2024-03-22 18:02:40.488 IST [21578] LOG:  checkpoint starting: shutdown immediate
2024-03-22 18:02:40.520 IST [21578] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.010 s, sync=0.005 s, total=0.035 s; sync files=3, longest=0.003 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/1554970, redo lsn=0/1554970
2024-03-22 18:02:40.540 IST [21575] LOG:  database system is shut down
2024-03-22 18:04:14.617 IST [21761] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 18:04:14.619 IST [21761] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 18:04:14.629 IST [21761] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 18:04:14.644 IST [21766] LOG:  database system was shut down at 2024-03-22 18:02:40 IST
2024-03-22 18:04:14.664 IST [21761] LOG:  database system is ready to accept connections
2024-03-22 18:09:14.715 IST [21764] LOG:  checkpoint starting: time
2024-03-22 18:09:14.764 IST [21764] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.018 s, sync=0.005 s, total=0.050 s; sync files=2, longest=0.005 s, average=0.003 s; distance=0 kB, estimate=0 kB; lsn=0/1554A58, redo lsn=0/1554A20
2024-03-22 19:00:27.603 IST [29204] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 19:00:27.606 IST [29204] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 19:00:27.619 IST [29204] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 19:00:27.641 IST [29209] LOG:  database system was interrupted; last known up at 2024-03-22 18:09:14 IST
2024-03-22 19:00:27.824 IST [29209] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-22 19:00:27.839 IST [29209] LOG:  redo starts at 0/1554A20
2024-03-22 19:00:27.839 IST [29209] LOG:  invalid record length at 0/1554B08: expected at least 24, got 0
2024-03-22 19:00:27.839 IST [29209] LOG:  redo done at 0/1554AD0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-22 19:00:27.848 IST [29207] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-22 19:00:27.873 IST [29207] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.010 s, sync=0.003 s, total=0.028 s; sync files=2, longest=0.002 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/1554B08, redo lsn=0/1554B08
2024-03-22 19:00:27.886 IST [29204] LOG:  database system is ready to accept connections
2024-03-22 19:58:18.287 IST [5754] FATAL:  lock file "postmaster.pid" already exists
2024-03-22 19:58:18.287 IST [5754] HINT:  Is another postmaster (PID 29204) running in data directory "/data/data/com.termux/files/home/nodeLogin/db"?
2024-03-22 19:58:29.806 IST [29204] LOG:  received fast shutdown request
2024-03-22 19:58:29.809 IST [29204] LOG:  aborting any active transactions
2024-03-22 19:58:29.815 IST [29204] LOG:  background worker "logical replication launcher" (PID 29212) exited with exit code 1
2024-03-22 19:58:29.817 IST [29207] LOG:  shutting down
2024-03-22 19:58:29.822 IST [29207] LOG:  checkpoint starting: shutdown immediate
2024-03-22 19:58:29.846 IST [29207] LOG:  checkpoint complete: wrote 0 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.007 s, sync=0.001 s, total=0.030 s; sync files=0, longest=0.000 s, average=0.000 s; distance=0 kB, estimate=0 kB; lsn=0/1554BB8, redo lsn=0/1554BB8
2024-03-22 19:58:29.863 IST [29204] LOG:  database system is shut down
2024-03-22 19:58:42.142 IST [5796] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 19:58:42.144 IST [5796] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 19:58:42.150 IST [5796] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 19:58:42.166 IST [5801] LOG:  database system was shut down at 2024-03-22 19:58:29 IST
2024-03-22 19:58:42.196 IST [5796] LOG:  database system is ready to accept connections
2024-03-22 19:58:44.992 IST [5796] LOG:  received fast shutdown request
2024-03-22 19:58:44.996 IST [5796] LOG:  aborting any active transactions
2024-03-22 19:58:45.003 IST [5796] LOG:  background worker "logical replication launcher" (PID 5804) exited with exit code 1
2024-03-22 19:58:45.003 IST [5799] LOG:  shutting down
2024-03-22 19:58:45.006 IST [5799] LOG:  checkpoint starting: shutdown immediate
2024-03-22 19:58:45.031 IST [5799] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.002 s, total=0.028 s; sync files=2, longest=0.002 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/1554C30, redo lsn=0/1554C30
2024-03-22 19:58:45.046 IST [5796] LOG:  database system is shut down
2024-03-22 19:59:13.119 IST [5880] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-22 19:59:13.121 IST [5880] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-22 19:59:13.129 IST [5880] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-22 19:59:13.144 IST [5885] LOG:  database system was shut down at 2024-03-22 19:58:45 IST
2024-03-22 19:59:13.162 IST [5880] LOG:  database system is ready to accept connections
2024-03-22 19:59:15.354 IST [5901] FATAL:  lock file "postmaster.pid" already exists
2024-03-22 19:59:15.354 IST [5901] HINT:  Is another postmaster (PID 5880) running in data directory "/data/data/com.termux/files/home/nodeLogin/db"?
2024-03-22 20:04:13.192 IST [5883] LOG:  checkpoint starting: time
2024-03-22 20:04:13.229 IST [5883] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.010 s, sync=0.002 s, total=0.038 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/1554D18, redo lsn=0/1554CE0
2024-03-22 23:55:57.756 IST [11076] FATAL:  could not open directory "pg_notify": No such file or directory
2024-03-22 23:55:57.762 IST [11076] LOG:  database system is shut down
2024-03-23 00:03:20.684 IST [11874] FATAL:  could not open directory "pg_notify": No such file or directory
2024-03-23 00:03:20.689 IST [11874] LOG:  database system is shut down
2024-03-23 00:17:42.823 IST [14914] FATAL:  could not open directory "pg_notify": No such file or directory
2024-03-23 00:17:42.828 IST [14914] LOG:  database system is shut down
2024-03-23 00:19:28.895 IST [15106] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 00:19:28.897 IST [15106] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 00:19:28.904 IST [15106] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 00:19:28.916 IST [15111] LOG:  database system was shut down at 2024-03-23 00:18:52 IST
2024-03-23 00:19:28.943 IST [15106] LOG:  database system is ready to accept connections
2024-03-23 00:20:59.444 IST [15309] ERROR:  could not open shared memory segment "pg_dynshmem/mmap.3892995486": No such file or directory
2024-03-23 00:21:29.093 IST [15106] LOG:  could not open file "postmaster.pid": No such file or directory
2024-03-23 00:21:29.093 IST [15106] LOG:  performing immediate shutdown because data directory lock file is invalid
2024-03-23 00:21:29.093 IST [15106] LOG:  received immediate shutdown request
2024-03-23 00:21:29.093 IST [15106] LOG:  could not open file "postmaster.pid": No such file or directory
2024-03-23 00:21:29.105 IST [15106] LOG:  could not remove shared memory segment "pg_dynshmem/mmap.3892995486": No such file or directory
2024-03-23 00:21:29.106 IST [15106] LOG:  could not remove shared memory segment "pg_dynshmem/mmap.2017522760": No such file or directory
2024-03-23 00:21:29.117 IST [15106] LOG:  database system is shut down
2024-03-23 00:22:19.058 IST [15485] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 00:22:19.060 IST [15485] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 00:22:19.068 IST [15485] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 00:22:19.079 IST [15490] LOG:  database system was shut down at 2024-03-23 00:21:57 IST
2024-03-23 00:22:19.101 IST [15485] LOG:  database system is ready to accept connections
2024-03-23 00:32:19.939 IST [20852] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 00:32:19.942 IST [20852] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 00:32:19.949 IST [20852] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 00:32:19.964 IST [20858] LOG:  database system was shut down at 2024-03-23 00:31:58 IST
2024-03-23 00:32:19.991 IST [20852] LOG:  database system is ready to accept connections
2024-03-23 00:36:41.467 IST [21459] ERROR:  role "dbuser" already exists
2024-03-23 00:36:41.467 IST [21459] STATEMENT:  CREATE USER dbuser WITH PASSWORD 'password' SUPERUSER;
2024-03-23 00:36:41.514 IST [21465] ERROR:  relation "users" already exists
2024-03-23 00:36:41.514 IST [21465] STATEMENT:  CREATE TABLE users (id BIGSERIAL,email VARCHAR(30),name VARCHAR(30),password TEXT);
2024-03-23 00:36:41.532 IST [20852] LOG:  received fast shutdown request
2024-03-23 00:36:41.538 IST [20852] LOG:  aborting any active transactions
2024-03-23 00:36:41.544 IST [20852] LOG:  background worker "logical replication launcher" (PID 20861) exited with exit code 1
2024-03-23 00:36:41.545 IST [20856] LOG:  shutting down
2024-03-23 00:36:41.548 IST [20856] LOG:  checkpoint starting: shutdown immediate
2024-03-23 00:36:41.603 IST [20856] LOG:  checkpoint complete: wrote 57 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.018 s, sync=0.027 s, total=0.058 s; sync files=29, longest=0.006 s, average=0.001 s; distance=250 kB, estimate=250 kB; lsn=0/154EAA0, redo lsn=0/154EAA0
2024-03-23 00:36:41.621 IST [20852] LOG:  database system is shut down
2024-03-23 00:37:06.058 IST [21533] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 00:37:06.060 IST [21533] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 00:37:06.069 IST [21533] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 00:37:06.082 IST [21542] LOG:  database system was shut down at 2024-03-23 00:36:41 IST
2024-03-23 00:37:06.111 IST [21533] LOG:  database system is ready to accept connections
2024-03-23 00:37:31.894 IST [21601] LOG:  could not receive data from client: Connection reset by peer
2024-03-23 00:38:04.634 IST [22048] LOG:  could not receive data from client: Connection reset by peer
2024-03-23 00:39:10.948 IST [23078] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 00:39:10.949 IST [23078] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 00:39:10.959 IST [23078] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 00:39:10.969 IST [23083] LOG:  database system was shut down at 2024-03-23 00:38:47 IST
2024-03-23 00:39:10.989 IST [23078] LOG:  database system is ready to accept connections
2024-03-23 00:44:10.651 IST [23081] LOG:  checkpoint starting: time
2024-03-23 00:44:10.793 IST [23081] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.118 s, sync=0.005 s, total=0.142 s; sync files=3, longest=0.003 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/155BC58, redo lsn=0/155BC20
2024-03-23 03:45:54.226 IST [15811] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 03:45:54.231 IST [15811] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 03:45:54.246 IST [15811] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 03:45:54.259 IST [15816] LOG:  database system was interrupted; last known up at 2024-03-23 00:44:10 IST
2024-03-23 03:45:54.396 IST [15816] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-23 03:45:54.407 IST [15816] LOG:  redo starts at 0/155BC20
2024-03-23 03:45:54.408 IST [15816] LOG:  invalid record length at 0/155BD08: expected at least 24, got 0
2024-03-23 03:45:54.408 IST [15816] LOG:  redo done at 0/155BCD0 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-23 03:45:54.414 IST [15814] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-23 03:45:54.440 IST [15814] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.002 s, total=0.028 s; sync files=2, longest=0.002 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/155BD08, redo lsn=0/155BD08
2024-03-23 03:45:54.455 IST [15811] LOG:  database system is ready to accept connections
2024-03-23 03:46:45.742 IST [16808] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 03:46:45.763 IST [16808] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 03:46:45.777 IST [16808] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 03:46:45.788 IST [16813] LOG:  database system was interrupted; last known up at 2024-03-23 03:45:54 IST
2024-03-23 03:46:45.935 IST [16813] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-23 03:46:45.939 IST [16813] LOG:  redo starts at 0/155BD80
2024-03-23 03:46:45.939 IST [16813] LOG:  invalid record length at 0/155BDB8: expected at least 24, got 0
2024-03-23 03:46:45.939 IST [16813] LOG:  redo done at 0/155BD80 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-23 03:46:45.946 IST [16811] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-23 03:46:45.969 IST [16811] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.002 s, total=0.026 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/155BDB8, redo lsn=0/155BDB8
2024-03-23 03:46:45.979 IST [16808] LOG:  database system is ready to accept connections
2024-03-23 03:51:45.507 IST [16811] LOG:  checkpoint starting: time
2024-03-23 03:51:45.746 IST [16811] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.212 s, sync=0.005 s, total=0.240 s; sync files=3, longest=0.004 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/155C0E0, redo lsn=0/155C0A8
2024-03-23 11:28:12.980 IST [10874] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 11:28:12.983 IST [10874] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 11:28:13.001 IST [10874] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 11:28:13.018 IST [10879] LOG:  database system was interrupted; last known up at 2024-03-23 03:51:45 IST
2024-03-23 11:28:13.192 IST [10879] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-23 11:28:13.201 IST [10879] LOG:  redo starts at 0/155C0A8
2024-03-23 11:28:13.201 IST [10879] LOG:  invalid record length at 0/155C190: expected at least 24, got 0
2024-03-23 11:28:13.201 IST [10879] LOG:  redo done at 0/155C158 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-23 11:28:13.208 IST [10877] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-23 11:28:13.234 IST [10877] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.003 s, total=0.029 s; sync files=2, longest=0.002 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/155C190, redo lsn=0/155C190
2024-03-23 11:28:13.246 IST [10874] LOG:  database system is ready to accept connections
2024-03-23 11:43:13.437 IST [10877] LOG:  checkpoint starting: time
2024-03-23 11:43:14.196 IST [10877] LOG:  checkpoint complete: wrote 8 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.721 s, sync=0.012 s, total=0.760 s; sync files=7, longest=0.006 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/155C528, redo lsn=0/155C4F0
2024-03-23 11:48:13.289 IST [10877] LOG:  checkpoint starting: time
2024-03-23 11:48:13.527 IST [10877] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.211 s, sync=0.006 s, total=0.239 s; sync files=3, longest=0.004 s, average=0.002 s; distance=1 kB, estimate=1 kB; lsn=0/155C938, redo lsn=0/155C900
2024-03-23 12:20:14.269 IST [19672] LOG:  could not receive data from client: Connection reset by peer
2024-03-23 12:33:15.492 IST [10877] LOG:  checkpoint starting: time
2024-03-23 12:33:15.736 IST [10877] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.210 s, sync=0.010 s, total=0.245 s; sync files=3, longest=0.008 s, average=0.004 s; distance=1 kB, estimate=1 kB; lsn=0/155CDC0, redo lsn=0/155CD88
2024-03-23 21:37:26.011 IST [7681] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-23 21:37:26.013 IST [7681] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-23 21:37:26.021 IST [7681] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-23 21:37:26.032 IST [7687] LOG:  database system was shut down at 2024-03-23 21:36:55 IST
2024-03-23 21:37:26.056 IST [7681] LOG:  database system is ready to accept connections
2024-03-23 21:40:13.278 IST [7681] LOG:  received fast shutdown request
2024-03-23 21:40:13.280 IST [7681] LOG:  aborting any active transactions
2024-03-23 21:40:13.287 IST [7681] LOG:  background worker "logical replication launcher" (PID 7690) exited with exit code 1
2024-03-23 21:40:13.287 IST [7684] LOG:  shutting down
2024-03-23 21:40:13.290 IST [7684] LOG:  checkpoint starting: shutdown immediate
2024-03-23 21:40:13.334 IST [7684] LOG:  checkpoint complete: wrote 47 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.011 s, sync=0.021 s, total=0.047 s; sync files=19, longest=0.005 s, average=0.002 s; distance=200 kB, estimate=200 kB; lsn=0/15423B0, redo lsn=0/15423B0
2024-03-23 21:40:13.349 IST [7681] LOG:  database system is shut down
2024-03-24 00:40:57.569 IST [25151] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-24 00:40:57.572 IST [25151] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-24 00:40:57.581 IST [25151] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-24 00:40:57.600 IST [25157] LOG:  database system was shut down at 2024-03-24 00:40:37 IST
2024-03-24 00:40:57.625 IST [25151] LOG:  database system is ready to accept connections
2024-03-24 00:41:12.767 IST [25174] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 00:45:57.698 IST [25155] LOG:  checkpoint starting: time
2024-03-24 00:46:02.180 IST [25155] LOG:  checkpoint complete: wrote 47 buffers (0.3%); 0 WAL file(s) added, 0 removed, 0 recycled; write=4.440 s, sync=0.016 s, total=4.482 s; sync files=19, longest=0.005 s, average=0.001 s; distance=200 kB, estimate=200 kB; lsn=0/1542420, redo lsn=0/15423E8
2024-03-24 00:53:30.403 IST [26224] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 00:54:50.612 IST [26339] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 02:55:49.080 IST [11146] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-24 02:55:49.085 IST [11146] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-24 02:55:49.091 IST [11146] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-24 02:55:49.106 IST [11152] LOG:  database system was shut down at 2024-03-24 02:55:17 IST
2024-03-24 02:55:49.129 IST [11146] LOG:  database system is ready to accept connections
2024-03-24 02:55:58.294 IST [11183] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 02:56:00.602 IST [11202] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 02:56:03.629 IST [11212] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 02:56:05.184 IST [11217] LOG:  could not receive data from client: Connection reset by peer
2024-03-24 03:00:49.186 IST [11150] LOG:  checkpoint starting: time
2024-03-24 03:00:49.527 IST [11150] LOG:  checkpoint complete: wrote 6 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.311 s, sync=0.010 s, total=0.341 s; sync files=5, longest=0.005 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/154EF20, redo lsn=0/154EEE8
2024-03-24 03:08:20.870 IST [12038] FATAL:  terminating connection due to unexpected postmaster exit
2024-03-24 03:09:15.463 IST [12992] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-24 03:09:15.474 IST [12992] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-24 03:09:15.497 IST [12992] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-24 03:09:15.522 IST [12997] LOG:  database system was interrupted; last known up at 2024-03-24 03:00:49 IST
2024-03-24 03:09:15.658 IST [12997] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-24 03:09:15.667 IST [12997] LOG:  redo starts at 0/154EEE8
2024-03-24 03:09:15.667 IST [12997] LOG:  invalid record length at 0/154EFD0: expected at least 24, got 0
2024-03-24 03:09:15.667 IST [12997] LOG:  redo done at 0/154EF98 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-24 03:09:15.675 IST [12995] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-24 03:09:15.703 IST [12995] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.011 s, sync=0.003 s, total=0.030 s; sync files=2, longest=0.002 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/154EFD0, redo lsn=0/154EFD0
2024-03-24 03:09:15.721 IST [12992] LOG:  database system is ready to accept connections
2024-03-24 03:14:16.483 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:14:16.619 IST [12995] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.109 s, sync=0.006 s, total=0.137 s; sync files=2, longest=0.004 s, average=0.003 s; distance=0 kB, estimate=0 kB; lsn=0/154F3A8, redo lsn=0/154F370
2024-03-24 03:18:19.119 IST [13006] ERROR:  syntax error at or near ")" at character 93
2024-03-24 03:18:19.119 IST [13006] STATEMENT:  create table love_life (
	id int,
	crush_name varchar(40),
	msg text,
	crush_email varchar(40),
	);
2024-03-24 03:18:43.273 IST [13006] ERROR:  relation "crush_name" does not exist at character 15
2024-03-24 03:18:43.273 IST [13006] STATEMENT:  select * from crush_name;
2024-03-24 03:19:16.719 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:19:18.974 IST [12995] LOG:  checkpoint complete: wrote 23 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=2.225 s, sync=0.008 s, total=2.255 s; sync files=20, longest=0.005 s, average=0.001 s; distance=114 kB, estimate=114 kB; lsn=0/156BE50, redo lsn=0/156BE18
2024-03-24 03:25:00.156 IST [13006] ERROR:  column "crush_name" of relation "users" does not exist at character 23
2024-03-24 03:25:00.156 IST [13006] STATEMENT:  INSERT INTO users (id,crush_name,msg,crush_email) VALUES (1,'Sonali','I love you Sona','sona@gmail.com');
2024-03-24 03:30:26.985 IST [13006] ERROR:  column reference "id" is ambiguous at character 48
2024-03-24 03:30:26.985 IST [13006] STATEMENT:  select * from love_life,users as details
	where id = 1;
2024-03-24 03:30:54.681 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:30:54.808 IST [12995] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.107 s, sync=0.006 s, total=0.127 s; sync files=2, longest=0.004 s, average=0.003 s; distance=0 kB, estimate=103 kB; lsn=0/156C018, redo lsn=0/156BFC8
2024-03-24 03:30:57.488 IST [13006] ERROR:  syntax error at or near "=" at character 61
2024-03-24 03:30:57.488 IST [13006] STATEMENT:  select * from love_life,users as details
	where users.id = 1 = love_life.id;
2024-03-24 03:31:07.093 IST [13006] ERROR:  invalid reference to FROM-clause entry for table "users" at character 48
2024-03-24 03:31:07.093 IST [13006] HINT:  Perhaps you meant to reference the table alias "details".
2024-03-24 03:31:07.093 IST [13006] STATEMENT:  select * from love_life,users as details
	where users.id = love_life.id;
2024-03-24 03:31:30.595 IST [13006] ERROR:  invalid reference to FROM-clause entry for table "users" at character 48
2024-03-24 03:31:30.595 IST [13006] HINT:  Perhaps you meant to reference the table alias "details".
2024-03-24 03:31:30.595 IST [13006] STATEMENT:  select * from love_life,users as details
	where users.id = 1 ;
2024-03-24 03:32:36.782 IST [13006] ERROR:  column love_life.crush does not exist at character 19
2024-03-24 03:32:36.782 IST [13006] STATEMENT:  select users.name,love_life.crush from love_life,users
	where users.id = 1;
2024-03-24 03:32:52.136 IST [13006] ERROR:  column love_life.crush does not exist at character 19
2024-03-24 03:32:52.136 IST [13006] STATEMENT:  select users.name,love_life.crush from love_life,users
	where users.id = 1;
2024-03-24 03:36:40.674 IST [13006] ERROR:  operator does not exist: bigint = character varying at character 103
2024-03-24 03:36:40.674 IST [13006] HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
2024-03-24 03:36:40.674 IST [13006] STATEMENT:  select users.name as boyfriend,love_life.crush_name as girlfriend from love_life,users
	where users.id = love_life.crush_name;
2024-03-24 03:40:54.966 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:40:55.211 IST [12995] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.216 s, sync=0.005 s, total=0.245 s; sync files=3, longest=0.003 s, average=0.002 s; distance=0 kB, estimate=92 kB; lsn=0/156C250, redo lsn=0/156C218
2024-03-24 03:43:03.231 IST [13006] ERROR:  relation "crush_name" does not exist at character 8
2024-03-24 03:43:03.231 IST [13006] STATEMENT:  UPDATE crush_name
	SET id = 2
	WHERE id = 1;
2024-03-24 03:45:54.313 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:45:54.749 IST [12995] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.412 s, sync=0.007 s, total=0.437 s; sync files=4, longest=0.003 s, average=0.002 s; distance=1 kB, estimate=83 kB; lsn=0/156C720, redo lsn=0/156C6E8
2024-03-24 03:46:19.462 IST [13006] ERROR:  invalid input syntax for type integer: "I love you Kuchu" at character 27
2024-03-24 03:46:19.462 IST [13006] STATEMENT:  UPDATE love_life
	SET id = 'I love you Kuchu'
	WHERE crush_name like 'So%';
2024-03-24 03:50:54.755 IST [12995] LOG:  checkpoint starting: time
2024-03-24 03:50:54.890 IST [12995] LOG:  checkpoint complete: wrote 2 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.109 s, sync=0.006 s, total=0.136 s; sync files=2, longest=0.005 s, average=0.003 s; distance=1 kB, estimate=75 kB; lsn=0/156CDF0, redo lsn=0/156CDB8
2024-03-24 04:00:55.234 IST [12995] LOG:  checkpoint starting: time
2024-03-24 04:00:55.469 IST [12995] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.210 s, sync=0.006 s, total=0.235 s; sync files=3, longest=0.004 s, average=0.002 s; distance=1 kB, estimate=68 kB; lsn=0/156D3A0, redo lsn=0/156D368
2024-03-24 04:01:27.704 IST [13006] ERROR:  missing FROM-clause entry for table "users" at character 8
2024-03-24 04:01:27.704 IST [13006] STATEMENT:  SELECT users.id, love_life.id, users.name, love_life.crush_name 
	FROM love_life
	WHERE love_life.id = 2;
2024-03-24 04:09:31.965 IST [19793] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-24 04:09:31.968 IST [19793] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-24 04:09:31.980 IST [19793] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-24 04:09:31.997 IST [19798] LOG:  database system was interrupted; last known up at 2024-03-24 04:00:55 IST
2024-03-24 04:09:32.178 IST [19798] LOG:  database system was not properly shut down; automatic recovery in progress
2024-03-24 04:09:32.189 IST [19798] LOG:  redo starts at 0/156D368
2024-03-24 04:09:32.189 IST [19798] LOG:  invalid record length at 0/156D930: expected at least 24, got 0
2024-03-24 04:09:32.190 IST [19798] LOG:  redo done at 0/156D8F8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2024-03-24 04:09:32.195 IST [19796] LOG:  checkpoint starting: end-of-recovery immediate wait
2024-03-24 04:09:32.215 IST [19796] LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.007 s, sync=0.004 s, total=0.022 s; sync files=3, longest=0.002 s, average=0.002 s; distance=1 kB, estimate=1 kB; lsn=0/156D930, redo lsn=0/156D930
2024-03-24 04:09:32.223 IST [19793] LOG:  database system is ready to accept connections
2024-03-24 04:09:54.435 IST [19820] FATAL:  database "database" does not exist
2024-03-24 04:11:12.049 IST [19831] ERROR:  syntax error at or near ")" at character 93
2024-03-24 04:11:12.049 IST [19831] STATEMENT:  create table love_life (
	id int,
	crush_name varchar(40),
	msg text,
	crush_email varchar(40),
	);
2024-03-24 04:11:26.610 IST [19831] ERROR:  relation "love_life" already exists
2024-03-24 04:11:26.610 IST [19831] STATEMENT:  create table love_life (
	id int,
	crush_name varchar(40),
	msg text,
	crush_email varchar(40));
2024-03-24 04:11:40.713 IST [19831] ERROR:  syntax error at or near ")" at character 93
2024-03-24 04:11:40.713 IST [19831] STATEMENT:  create table love_life (
	id int,
	crush_name varchar(40),
	msg text,
	crush_email varchar(40),
	);
2024-03-24 04:13:04.127 IST [20161] ERROR:  role "dbuser" already exists
2024-03-24 04:13:04.127 IST [20161] STATEMENT:  CREATE USER dbuser WITH PASSWORD 'password' SUPERUSER;
2024-03-24 04:13:04.171 IST [20163] ERROR:  relation "users" already exists
2024-03-24 04:13:04.171 IST [20163] STATEMENT:  CREATE TABLE users (id BIGSERIAL,email VARCHAR(30),name VARCHAR(30),password TEXT);
2024-03-24 04:13:04.204 IST [20165] ERROR:  relation "love_life" already exists
2024-03-24 04:13:04.204 IST [20165] STATEMENT:  create table love_life (id int,crush_name varchar(40),msg text,crush_email varchar(40));
2024-03-24 04:13:04.220 IST [19793] LOG:  received fast shutdown request
2024-03-24 04:13:04.224 IST [19793] LOG:  aborting any active transactions
2024-03-24 04:13:04.225 IST [19831] FATAL:  terminating connection due to administrator command
2024-03-24 04:13:04.230 IST [19793] LOG:  background worker "logical replication launcher" (PID 19801) exited with exit code 1
2024-03-24 04:13:04.231 IST [19796] LOG:  shutting down
2024-03-24 04:13:04.234 IST [19796] LOG:  checkpoint starting: shutdown immediate
2024-03-24 04:13:04.262 IST [19796] LOG:  checkpoint complete: wrote 13 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.008 s, sync=0.010 s, total=0.032 s; sync files=14, longest=0.003 s, average=0.001 s; distance=50 kB, estimate=50 kB; lsn=0/157A2A8, redo lsn=0/157A2A8
2024-03-24 04:13:04.273 IST [19793] LOG:  database system is shut down
2024-03-24 04:24:14.702 IST [21189] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-03-24 04:24:14.704 IST [21189] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-03-24 04:24:14.715 IST [21189] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-03-24 04:24:14.729 IST [21194] LOG:  database system was shut down at 2024-03-24 04:13:04 IST
2024-03-24 04:24:14.750 IST [21189] LOG:  database system is ready to accept connections
2024-03-24 04:29:26.778 IST [21192] LOG:  checkpoint starting: time
2024-03-24 04:29:27.021 IST [21192] LOG:  checkpoint complete: wrote 5 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.217 s, sync=0.008 s, total=0.244 s; sync files=4, longest=0.004 s, average=0.002 s; distance=0 kB, estimate=0 kB; lsn=0/157A390, redo lsn=0/157A358
2024-04-06 12:41:16.544 IST [25554] LOG:  starting PostgreSQL 16.2 on aarch64-unknown-linux-android, compiled by Android (10552028, +pgo, +bolt, +lto, -mlgo, based on r487747d) clang version 17.0.2 (https://android.googlesource.com/toolchain/llvm-project d9f89f4d16663d5012e5c09495f3b30ece3d2362), 64-bit
2024-04-06 12:41:16.546 IST [25554] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2024-04-06 12:41:16.555 IST [25554] LOG:  listening on Unix socket "/data/data/com.termux/files/usr/tmp/.s.PGSQL.5432"
2024-04-06 12:41:16.910 IST [25559] LOG:  database system was shut down at 2024-04-06 12:40:42 IST
2024-04-06 12:41:16.943 IST [25554] LOG:  database system is ready to accept connections
2024-04-06 12:46:17.465 IST [25557] LOG:  checkpoint starting: time
2024-04-06 12:46:19.154 IST [25557] LOG:  checkpoint complete: wrote 19 buffers (0.1%); 0 WAL file(s) added, 0 removed, 0 recycled; write=1.632 s, sync=0.021 s, total=1.690 s; sync files=12, longest=0.008 s, average=0.002 s; distance=34 kB, estimate=34 kB; lsn=0/15239A0, redo lsn=0/1523968
